packages/uui-dialog-layout/lib/uui-dialog-layout.story.ts

```diff
@@ -0,0 +1,20 @@
+import '.';
+
+import { Story } from '@storybook/web-components';
+import { html } from 'lit-html';
+
+export default {
+  id: 'uui-dialog-layout',
+  title: 'Displays/Dialog Layout',
+  component: 'uui-dialog-layout',
+  parameters: {
+    docs: {
+      source: {
+        code: `<uui-dialog-layout></uui-dialog-layout>`,
+      },
+    },
+  },
+};
+
+export const Overview: Story = () =>
+  html`<uui-dialog-layout></uui-dialog-layout>`;
```

Comment:
Could we add a comment and a link to view the dialog story for an example of this?

Reply:
We might also try out to put the dialog-layout component as a child to dialog in Storybook.

Or maybe having a folder for dialog and then having dialog and dialog-layout under that.. that's the approach on the toast notification, so that is properly nicest when we do the same across components :-)